cluster-autoscaler/cloudprovider/externalgrpc/externalgrpc_cloud_provider.go

```diff
@@ -178,6 +184,10 @@ func (m *pricingModel) PodPrice(pod *apiv1.Pod, startTime time.Time, endTime tim
 		EndTime:   &end,
 	})
 	if err != nil {
+		st, ok := status.FromError(err)
+		if ok && st.Code() == codes.Unimplemented {
+			return 0, cloudprovider.ErrNotImplemented
```

Comment:
why zero for the price? I mean we raise the error anyway sso it doesn't matter, but zero could potentially be a real price. Maybe -1 makes more sense? To further raise eyebrows this isn't successful data from the pricing model?

Reply:
> why zero for the price? 
no particular reason, it is a standard practice to return the default value for the type when the value does not matter
> I mean we raise the error anyway sso it doesn't matter
exactly
> but zero could potentially be a real price
also -1, and it is a even better price than 0 if it is used by mistake :wink: .
I think the point is that no matter the return value, errors should be handled, and the cluster autoscaler core does it, not doing that is a logic error by the caller.